provider/resource_rediscloud_essentials_database.go

```diff
@@ -664,7 +762,6 @@ func resourceRedisCloudEssentialsDatabaseUpdate(ctx context.Context, d *schema.R
 		// ClientTlsCertificates
 		// Password
 		// Alerts
-		// Modules
```

Comment:
Do we actually need any of these commented lines above?

Reply:
No, they were more for me to say "I'm not writing them here but they will be written to the object later, with a little bit of extra logic"